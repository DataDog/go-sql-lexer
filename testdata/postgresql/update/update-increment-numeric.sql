UPDATE accounts SET balance = balance + 100.0 WHERE user_id = 4;
