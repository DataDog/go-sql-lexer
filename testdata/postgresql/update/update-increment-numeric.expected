[
    {
        "expected": "UPDATE accounts SET balance = balance + ? WHERE user_id = ?",
        "statement_metadata": {
            "size": 14,
            "tables": [
                "accounts"
            ],
            "commands": [
                "UPDATE"
            ],
            "comments": [],
            "procedures": []
        }
    }
]